server/dev-db-init.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# Get environment variables
+source $(dirname "${BASH_SOURCE[0]}")/.env.dev
+POSTGRES_USER=postgres
```

Comment:
Minor bash convention comment, capital variable names are usually for environment variables. Local variables (meaning they aren't turned in to env vars with `export <variable name>`) should use lower case names :+1:

Reply:
Great to know!  I've removed the variable completely and just added postgres inline.  (Or is it better practice to keep this as a variable?)